src/components/Menus/WalletMenu/WalletCard.tsx

```diff
@@ -74,6 +78,19 @@ export const WalletCard = ({ account }: WalletCardProps) => {
     }
   };
 
+  const handleScanButton = () => {
+    account.chainId && closeAllMenus();
+    const url = `${JUMPER_SCAN_PATH}/wallet/${account.address}`;
+
+    trackEvent({
+      category: TrackingCategory.WalletMenu,
+      action: TrackingAction.OpenJumperScan,
+      label: 'open-jumper-scan-wallet',
+      disableTrackingTool: [EventTrackingTool.ARCx, EventTrackingTool.Cookie3],
+    });
+    window.open(url, '_self');
```

Comment:
@tcheee @oktapodia This `window.open` allows to navigate to the scan page with wallet-view. I had issues with `router.push`. Wdyt about this workaround?

Reply:
What issues did you get? window.open won't do anything in most of the borwser/users as it is considered intrusive for users